NuGet.Config

```diff
@@ -9,8 +9,4 @@
   <activePackageSource>
     <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
   </activePackageSource>
-  <packageRestore>
-    <add key="enabled" value="False" />
-    <add key="automatic" value="False" />
-  </packageRestore>
```

Comment:
Not technically required for this PR but now that we have some tooling, it kinda works. I'll probably revert before merging.